packages/docs/content/components/alert.mdx

```diff
@@ -250,13 +250,94 @@ A custom heading value can be sent for an analytics event by using the prop `ana
 
 ### Accessibility
 
+<ThemeContent onlyThemes={['healthcare']}>
+
+If dynamically loaded, be mindful of when the alert displays on the page. For example, an alert appearing on a page due to an error or progressive disclosure. Where and how the content appears visually should align with how and when a screen reader user hears the content. Errors need immediate attention, but content through progressive disclosure may not.
+
+- Users who are viewing the content need it to be placed in an effective location and to draw attention even when CSS is turned off. Content can help here too, using word choices that make the importance of the message clear.
+- Users who hear this content through assistive technology, such as a screen readers, will need to be alerted through the backend code effectively at the right time.
```

Comment:
```suggestion
- Users who hear this content through assistive technology, such as a screen readers, will need to be alerted at the right time, and that can be managed with code.
```

Trying to help this make more sense. It doesn't necessarily take extra "backend code" to do this, but it may require some code tweaking if the defaults won't cut it.